source/documentation/4.0/rspec-rails/file.Changelog.html

```diff
@@ -58,7 +58,18 @@
         <div class="clear"></div>
       </div>
 
-      <div id="content"><div id='filecontents'><h3>4.0.0 / 2020-03-24</h3>
+      <div id="content"><div id='filecontents'><h3>Development</h3>
+
+<p><a href="https://github.com/rspec/rspec-rails/compare/v4.0.0...4-0-development">Full Changelog</a></p>
+
+<p>Bug Fixes:</p>
+
+<ul>
+<li>Remove warning when calling <code>driven_by</code> in system specs. (Aubin Lorieux, #2302)</li>
+<li>Fix comparison of times for <code>#at</code> in job matchers. (Jon Rowe, Markus Doits, #2304)</li>
+</ul>
+
+<h3>4.0.0 / 2020-03-24</h3>
```

Comment:
This shouldn't be here until released, might need to run after a `git checkout v4.0.0`

Reply:
This is what I do but the release was on [`d810df5`](https://github.com/rspec/rspec-rails/releases/tag/v4.0.0) and the rake command _pull_ to the last commit from `v4.0.0`. It is the normal behavior of the command. For this time I will skip the `git pull` and stick to the commit from the release.